menas/src/main/scala/za/co/absa/enceladus/menas/repositories/MonitoringMongoRepository.scala

```diff
@@ -67,7 +67,7 @@ class MonitoringMongoRepository @Autowired()(mongoDb: MongoDatabase)
         Document(
           s"""{ $$match: {
              |    informationDateCasted: {$$gte: ISODate("${startDate}T00:00:00.0Z"),
-             |      $$lte: ISODate("${endDate}T00:00:00.0Z") }
+             |      $$lte: ISODate("${endDate}T23:59:59.9Z") }
```

Comment:
Why this change?

Reply:
To include the infodates where time is not set to 00:00:00 UTC into the search results. 
The time is not supposed to be included at all, however, I came across example where the infodate is always 'YYYY-MM-DD 23:59:59.590' ...